terraform/transition.fec.gov.tf

```diff
@@ -8,14 +8,20 @@
 # and because you can't CNAME the root domain, Savvis/CenturyLink are 
 # pointing the root at an FEC server that HTTP redirects to www.fec.gov
 
-
 resource "aws_route53_zone" "transition_gov_us_zone" {
   name = "transition.fec.gov"
   tags {
     Project = "dns"
   }
 }
 
+resource "aws_route53_zone" "www_fec_gov" {
```

Comment:
Yes, because terraform refers to resources by their type and name, so there's no need to include the type in the name. We'd refer to this resource using `aws_route53_zone.www_fec_gov`.